_data/data.js

```diff
@@ -239,14 +241,25 @@ const downloadImage = async function (url, file) {
     });
 
     response.data.on('end', () => {
-      console.log(`Image received: ${path} (${hash.digest('hex')})`);
+      hashCode = hash.digest('hex');
+      console.log(`Image received: ${path} (${hashCode})`);
     });
     writer.on('finish', () => {
       console.timeEnd(label);
+      const hashFileName = `${hashCode}${extension}`;
+      const newPath = Path.join(IMAGE_DIRECTORY, hashFileName);
+      const newVirtualPath = Path.join(IMAGE_PATH, hashFileName);
```

Comment:
Maybe `hashPath` and `hashVirtualPath`? I'm trying to make the names more concrete and avoid using "new" that's ambiguous.